opsdroid/connector/matrix/connector.py

```diff
@@ -323,6 +367,7 @@ async def _send_message(self, message):
             self._get_formatted_message_body(
                 message.text, msgtype=self.message_type(message.target)
             ),
+            ignore_unverified_devices=True,
```

Comment:
leaving this as an instance variable is probably a good idea. I can see that this will likely become a flag in the future. Could be private (`_`)?